paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOptionFactory.kt

```diff
@@ -156,4 +161,10 @@ internal class PaymentOptionFactory @Inject constructor(
             else -> resourceId
         }
     }
+
+    companion object {
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
This should be able to be internal as well

Reply:
ah thanks! Updated